_posts/2016-05-26-blm-utah-offers-new-recreation-maps-for-mobile-devices.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+status: publish
+published: true
+title: 'BLM-Utah Offers New Recreation Maps for Mobile Devices'
+author:
+  display_name: Kimberly Finch, Public Affairs Specialist, Bureau of Land Management-Utah
+  email: kfinch@blm.gov
+date: 2016-05-26 16:10:01
+categories:
+- Guest Blog
+tags:
+- BLM
+- GeoPDF
+- Recreation
+- Great Basin Institute 
+---
+<div class="caption"><a href ="{{ "/images/BLMgeopdfs1.jpg " | prepend: site.baseurl }}"><img src="{{ "/images/BLMgeopdfs1_small.jpg" | prepend: site.baseurl }}" alt="Using the GeoPDF Maps in the Field" /></a><p class="caption-text">Great Basin Institute Associate and map designer Jonathan Jew shows Jaydon Mead from the Price Field Office how to use the Geo PDF maps in the field.</p></div>
```

Comment:
Do not start a post with a caption or image otherwise the snippet in the news feed is blank. move this down a paragraph.